biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.spec.ts

```diff
@@ -51,7 +50,8 @@ describe('tests for the BioEnergySource', () => {
       efficiency = 0.125,
       latitude = 0,
       area = 10,
-      date = new Date('07/20/2020');
+      date = new Date();
+    date.setHours(2);
```

Comment:
This is 2AM?  The test description mentions you're getting power output during the day

Reply:
My bad, that should not be there